packages/web/src/scss/components/Collapse/index.html

```diff
@@ -167,6 +167,42 @@ <h2 class="docs-Heading">Collapse with hide trigger when expanded </h2>
 
 </section>
 
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default with helper class for changing content inside toggle</h2>
+
+  <p>
+    Ante rhoncus, tristique nunc et elit id in ex sem consectetur id, consectetur et elit nibh sed ac. Sed ac convallis, sem lobortis convallis et nisi sapien in ex sem ullamcorper
+    iaculis, aliquet sem lobortis pellentesque sit amet. Congue nulla congue, porttitor quis orci quam elit rutrum scelerisque nibh maximus, felis a integer maximus praesent.
+    Metus varius, at sem dui vivamus elementum luctus nisl sit amet dolor sit amet, mauris et iaculis nec fermentum. Augue sapien, nisl vel purus nullam odio morbi a fusce vitae
+    mattis vitae, posuere tristique bibendum aliquam. In donec dolor ut, imperdiet quam fermentum molestie vulputate scelerisque ac nec, tortor mi orci sollicitudin elementum.
+  </p>
+  <div
+    id="CollapseExampleV"
+    class="Collapse"
+  >
+    <div class="Collapse__content">
+      Aliquam varius, consequat posuere a lacinia mauris eu tellus condimentum ut id ante, accumsan vehicula nulla neque. Mauris mi orci, in donec nullam odio leo sapien et vehicula nunc a
+      lacinia, fermentum arcu ullamcorper posuere. Mauris euismod, ac nec ante fermentum praesent nisi commodo neque placerat, vivamus dui et tempus pulvinar suspendisse. Porttitor eget,
+      sollicitudin hendrerit bibendum nulla aliquam sit amet leo vitae, eget consectetur diam a vestibulum. Adipiscing lorem ipsum, arcu condimentum posuere semper morbi condimentum dui,
+      bibendum nunc aenean facilisis. Phasellus euismod, donec sem odio ligula praesent finibus nibh convallis, tristique aliquam sed id tortor sem lobortis.
+    </div>
+  </div>
+  <div class="mb-400">
+    <button
```

Comment:
Is the button the only way to do this behavior? What about an anchor with a button role? I saw many cases where link was used to show more or less. Could we add also an example of this kind?

Reply:
@literat @crishpeen @adamkudrna So should I delete it now?

Reply:
**For the record:** No, keep it 🙂.

We decided to leave it here because we need the link-like buttons and there is currently no other way to achieve that.